scripts/jules_gh_feedback.sh

```
@@ -110,19 +112,19 @@ for res in results:
     cpu = res.get("cpu_percentage", "0.0%")
     mem = str(res.get("memory_usage_bytes", "0"))
     err = res.get("error_summary", "") or "-"
+    logs = res.get("logs", "") or "No output logged."
+
+    # Determine code fence length dynamically to prevent logs containing triple backticks from terminating the block
+    fence = "```"
+    while fence in logs:
+        fence += "`"
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)